### PR TITLE
Typescript typings for grid stuff. Also, I was missing the Pagination…

### PR DESCRIPTION
**Pull request by roblevintennis**
No Asana...this is counterpart to https://github.com/getethos/ethos/pull/1903 which requires the Data Grid and Pagination stuff is exported and also Typescript typed.

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -1,6 +1,6 @@
 {
   "name": "ethos-design-system",
-  "version": "1.1.42",
+  "version": "1.1.45",
```
Comment (roblevintennis) [inline package.json]: The reason for bump from 42 to 45 is I've found yarn link and local development flaky, so I just prefer to bump the EDS patch versions until it's working properly ;)

File: src/components/index.js [hunk 2]
```diff
@@ -10,7 +10,7 @@ export { COLORS } from './Colors'
 export { EmailInput } from './EmailInput'
 export { Faq } from './Faq'
 export { Form } from './Form'
-export { Grid, Row, Column, useGridSorting } from './Grid'
+export { Grid, Pagination, Row, Column, useGridSorting } from './Grid'
```
Comment (ankur-ethos) [inline src/components/index.js]: is this an unused import?

File: src/components/index.d.ts [hunk 3]
```diff
@@ -286,6 +286,50 @@ export declare namespace RadioButtonGroup {
...
+export declare const useGridSorting: (
+  rows: any,
+  columns: any
+) => {
+  rowsRefs: any
+  columnRefs: any[]
+  sortedRows: any
+  compareBy: (
+    key: any,
+    sortMethod?: (a: any, b: any) => 1 | 0 | -1
```
Comment (ankur-ethos) [inline src/components/index.d.ts]: what's the default implementation of `sortMethod`?

File: src/components/index.d.ts [hunk 4]
```diff
@@ -286,6 +286,50 @@ export declare namespace RadioButtonGroup {
...
+    renderCallback: any
+  }): JSX.Element
+  propTypes: {
+    fetchPageCallback: any
+    renderCallback: any
+  }
+  displayName: string
+}
+export declare const useGridSorting: (
+  rows: any,
```
Comment (ankur-ethos) [inline src/components/index.d.ts]: just for my understanding, do these really need to be `any`?

Comment (roblevintennis) [reply src/components/index.d.ts]: Probably not but I plan to be iterative and refine these types further

File: src/components/index.js (hunk 2 above)
Comment (roblevintennis) [reply src/components/index.js]: It’s an export that is now needed now that I wanted to consume it from outside of EDS.

**Comment by roblevintennis** [comment]
Thanks guys. I agree we can do better in terms of typing and I definitely plan to continue improving in the area of EDS typings. I’m gonna merge this in spirit of being iterative and continue to refine over next couple/few weeks.

**Pull request merged**